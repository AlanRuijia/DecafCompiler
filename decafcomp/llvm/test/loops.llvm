; ModuleID = 'C'

define void @f() {
entry:
  %x = alloca i32
  %i = alloca i32
  store i32 100, i32* %size
  ret void
}
