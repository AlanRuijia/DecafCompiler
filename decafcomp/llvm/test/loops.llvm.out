semantic error: Undefined variable:leftPtr.
